src/main/java/mate/academy/Main.java

```diff
@@ -6,18 +6,23 @@
 import mate.academy.service.AuthenticationServiceImpl;
 import mate.academy.service.OrderService;
 import mate.academy.service.OrderServiceImpl;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
 
 public class Main {
+    private static final Logger logger = LogManager.getLogger(Main.class);
+    private static final AuthenticationService authenticationService =
+            new AuthenticationServiceImpl();
+    private static final OrderService orderService = new OrderServiceImpl();
```

Comment:
```suggestion
    private static final AuthenticationService authenticationService =
            new AuthenticationServiceImpl();
    private static final OrderService orderService = new OrderServiceImpl();
```
you fon`t need this fields